addons/mrp/static/src/components/bom_overview_line/mrp_bom_overview_line.js

```diff
@@ -170,9 +175,19 @@ export class BomOverviewLine extends Component {
     }
 
     get statusBackgroundClass() {
```

Comment:
I think those conditions are overly complicated. Here are the possible scenarios for `availability_state` / `stock_avail_state`:

| Description | `availability_state` | `stock_avail_state` | Expected color |
|-|-|-|-|
| Enough in stock | available | available | green |
| Enough in stock in the future | expected | expected | orange |
| Not enough in stock but route | estimated | unavailable | red |
| Not enough in stock & no route | unavailable | unavailable | red |